subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -191,13 +191,27 @@ uint8_t ll_adv_aux_sr_data_set(uint8_t handle, uint8_t op, uint8_t frag_pref, ui
 	 */
 	pri_pdu_prev = lll_adv_data_peek(lll);
 	if (pri_pdu_prev->type != PDU_ADV_TYPE_EXT_IND) {
+		if ((op != BT_HCI_LE_EXT_ADV_OP_COMPLETE_DATA) || (len > 31)) {
```

Comment:
Use `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX` instead of 31?

Reply:
this is explicitly defined by spec as 31 bytes (since it's max size of legacy scan response). I did not find any `#define` for this so just used hardcoded value.

Reply:
my fault, I missed the `!= PDU_ADV_TYPE_EXT_IND`